webtau-core/src/main/java/org/testingisdocumenting/webtau/reporter/WebTauTest.java

```diff
@@ -17,17 +17,22 @@
 
 package org.testingisdocumenting.webtau.reporter;
 
-import org.testingisdocumenting.webtau.reporter.stacktrace.StackTraceCodeEntry;
-import org.testingisdocumenting.webtau.reporter.stacktrace.StackTraceUtils;
-import org.testingisdocumenting.webtau.time.Time;
-import org.testingisdocumenting.webtau.utils.FileUtils;
+import static org.testingisdocumenting.webtau.reporter.TestStatus.Errored;
```

Comment:
My IntelliJ's Google-style insisted on this import order:
1. import static
1. java
1. others

Many other files are clearly not following this ordering. Do we have a style guide to import that I overlooked?

Reply:
we don't. I use whatever default in IDEA. I don't bother too much tbh.